src/main/java/seedu/taassist/logic/commands/DeletecCommand.java

```diff
@@ -41,6 +45,18 @@ public CommandResult execute(Model model) throws CommandException {
                     model.getModuleClassList()));
         }
 
+        List<Student> students = model.getFilteredStudentList();
```

Comment:
Shouldn't we delete from the entire `model` list? Instead of just the filtered model list.

Reply:
Good concern. However, the fix might be slightly breaking abstraction, we might need to look into the implementation in the future. Will get it fixed in the next commit.